### DISCUSSION TITLE
3.1.4

### DISCUSSION THREAD

**Discussion by LabhanshAgrawal** [vercel/hyper#6006] [Announcements]
V3.1.4

|   OS    | Installer |                                                                                                 |                                                                                                  |
| :-----: | :-------: | :---------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------: |
|   Mac   |    dmg    | [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/Hyper-3.1.4-mac-x64.dmg) (Intel) | [arm64](https://github.com/vercel/hyper/releases/download/v3.1.4/Hyper-3.1.4-mac-arm64.dmg) (M1) |
|  Linux  |    deb    |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/hyper_3.1.4_amd64.deb)      |     [arm64](https://github.com/vercel/hyper/releases/download/v3.1.4/hyper_3.1.4_arm64.deb)      |
|         |    rpm    |     [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/hyper-3.1.4.x86_64.rpm)      |    [arm64](https://github.com/vercel/hyper/releases/download/v3.1.4/hyper-3.1.4.aarch64.rpm)     |
|         | AppImage  |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/Hyper-3.1.4.AppImage)       |   [arm64](https://github.com/vercel/hyper/releases/download/v3.1.4/Hyper-3.1.4-arm64.AppImage)   |
|         |   snap    |     [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/hyper_3.1.4_amd64.snap)      |                                                                                                  |
| Windows |    exe    |      [x64](https://github.com/vercel/hyper/releases/download/v3.1.4/Hyper-Setup-3.1.4.exe)      |                                                                                                  |

## Changelog
- Focus the terminal after closing search (a3ca91d593770b0d12741a351f99ca15128dc390)
- Support notarization on macOS (#5994)
- Re-add default tab switching keymaps on windows (#5981)
- use `alt+f` as the default keymap for hamburger menu (#6005)

**Full Changelog**: https://github.com/vercel/hyper/compare/v3.1.3...v3.1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/hyper/releases/tag/v3.1.4'>3.1.4</a>.</em>

**Comment by rx-dev** [comment]
The version 3.1.4 message has arrive red message box won't disappear on Hyper restart, etc. This has never happened in the past. I'm not sure if this is a bug or a user error, but it's a little annoying. Any help would be appreciated, thanks!
![image](https://user-images.githubusercontent.com/20506548/136858275-cebadc69-cbfb-4491-be04-a3293e6c318e.png)


Edit: Nevermind, it decided to resolve itself after like 3 days lol

**Reply by LabhanshAgrawal** [reply]
Which platform and hyper version are you using?

**Reply by abheist** [reply]
same.. and getting the following error while installing.
<img width="417" alt="Screenshot 2021-10-12 at 8 23 29 PM" src="https://user-images.githubusercontent.com/5029660/136979559-146c05c9-e983-4267-9497-053742d5de22.png">
.

**Reply by LabhanshAgrawal** [reply]
Also you could have clicked the x button at the end to close this notification

**Reply by LabhanshAgrawal** [reply]
> same.. and getting the following error while installing. <img alt="Screenshot 2021-10-12 at 8 23 29 PM" width="417" src="https://user-images.githubusercontent.com/5029660/136979559-146c05c9-e983-4267-9497-053742d5de22.png"> .

can you try and redownload maybe?
also you can get updated automatically also if you are facing issues with the downloaded file.

**Reply by abheist** [reply]
I redownloaded and installed it many times. It didn't worked!

**Reply by dwhelan-rv** [reply]
oooooooh theres an X button.  Contrast ;)
